cypress/support/commands.ts

```diff
@@ -17,6 +17,12 @@ declare global {
 }
 
 Cypress.Commands.add('loadPage', (pageName) => {
+  cy.intercept('/samples/**/*', { middleware: true }, (req) => {
+    req.on('before:response', (res) => {
+      // force all API responses to not be cached
+      res.headers['cache-control'] = 'no-store';
+    });
+  });
```

Comment:
This ensures network requests aren't being cached in tests, which makes waiting for network intercepts more reliable. I added the same block to the `loadPdf()` function below.